pkg/store/project/project.go

```diff
@@ -78,3 +78,15 @@ func (s *store) Exists(db *gorm.DB, id string) (bool, error) {
 	query := db.Raw("SELECT EXISTS (SELECT * FROM projects WHERE id = ?) as result", id)
 	return record.Result, query.Scan(&record).Error
 }
+
+// One get 1 project by id
+func (s *store) One(db *gorm.DB, id string) (*model.Project, error) {
+	var project *model.Project
+	return project, db.Where("id = ?", id).
+		Preload("Members", "deleted_at IS NULL and left_date IS NULL AND status = ?", model.ProjectMemberStatusActive).
```

Comment:
Not only `active` member. We have 2 types of `active` member. `active` and `on-boarding`